common/signergroupops.go

```diff
@@ -476,16 +465,19 @@ func (mdb *MusicDB) CheckIfProcessComplete(tx *sql.Tx, sg *SignerGroup) (bool, s
 			if err != nil {
 				log.Printf("CheckIfProcessIsComplete: Error from db.Prepare(%s): %v",
 								      sqlq, err)
+				return false, fmt.Sprintf("Error from mdb.Prepare(%s): %v", sqlq, err), err
```

Comment:
What is the returning string from this function?

Reply:
En felutskrift. I grunden är strängen avsedd att vara feedback tillbaka till en användare. Men om någonting gick fel i en Prepare() så är det ju snarare frågan om en bugg i koden, dvs meddelandet används här mest som feedback till mig utan att behöva gräva i alla loggar.

Reply:
Då skulle jag printa ut något istället, eller lägga på det på err variabeln. Allting som returneras från en funktion är till för att den som anropat funktionen ska använda det.

Reply:
I see your point :-)